xml/installation-installation-sles-sles_preinstall.xml

```diff
@@ -17,14 +17,11 @@
  <section>
   <title>Sample iptables rules must be removed on &slsa;</title>
   <para>
-   <!-- FIXME: Red Hat content in SLES section. iptables-services package
-   does not seem to exist (at least on openSUSE). - sknorr, 2018-01-29 -->
    &kw-hos-phrase; uses iptables to secure access to &lcm; network
-   interfaces and on &rh; this requires the package
-   <literal>iptables-services</literal> to be installed. The package will
-   provide sample iptables configurations for IPv4 and IPv6 if none existed
-   before. This sample configuration is inappropriate for &kw-hos; operation and
-   the node will not be able to run &productname; with these rules installed.
+   interfaces. Any iptables configuration packages previously installed must
+   be removed. Some sample configurations are inappropriate for &kw-hos;
+   operation. The node will not be able to run &productname; with these rules
+   installed.
```

Comment:
This sounds too much like muddying down the information to me. Have you checked whether there are equivalent packages in SLE? If there are none, I think this entire section can be removed.

Reply:
@sknorr 
Per Dirk Mueller:
> ardana is handling iptables on its own on SLES and installs/deinstalls stuff as necessary.

So this entire section is unnecessary. I commented out the xi:include in
installation-installation-preinstall_checklist.xml. Not sure what to do with
installation-installation-sles-sles_preinstall.sml ... eliminate completely?

Reply:
I think a `git rm` would be best, yes. Thanks for checking!

Reply:
@sknorr
done
from feature/installation_remove-rhel2 
git rm xml/installation-installation-sles-sles_preinstall.xml
I don't see those changes in this pull request. I'm not going to risk messing up something. I'll just leave this until you say differently.

Reply:
Did you commit and push those changes?